.pylintrc

```diff
@@ -151,7 +151,9 @@ disable=raw-checker-failed,
         suppressed-message,
         useless-suppression,
         deprecated-pragma,
-        use-symbolic-message-instead
+        use-symbolic-message-instead,
+        invalid-name,
```

Comment:
It is possible I think to set this up to check for either snake_case or a single character string, which I think would cover all of our variables, via a regex. Although I'm not sure why the single characters are being complained about by pylint as I think single characters *are* valid snake case.

Reply:
I've added a regex to catch snake_case of 1 or more character variable names now